### PR TITLE
feat: geofence the US

### PR DESCRIPTION
**Pull request by johnletey**

### CODE REVIEW

**Comment by johnletey** [comment]
@t8 Can you work on the content of the `/usa` page 🙌🏻

**Review by t8 — DISMISSED** [review]
LGTM! Once we implement Max' design, this should be good. Once that is done, don't merge this yet. Waiting on a few things before we do that.

Thanks @johnletey!

File: src/routes/usa.svelte [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  <title>Verto — USA</title>
+</svelte:head>
+
+<NavBar />
+<div class="usa">
+  <div class="container">
+    <div class="text">
+      <h1>
+        4 <img
+          src="https://twemoji.maxcdn.com/v/latest/svg/1f1fa-1f1f8.svg"
```
Comment (t8) [inline src/routes/usa.svelte:30]: We should serve this locally

Comment (johnletey) [reply src/routes/usa.svelte]: @t8 Added in https://github.com/useverto/verto/pull/84/commits/c10834805be0b19b13ba88f5c4cd54be097d2473 😄

**Review by t8 — APPROVED** [review]
LGTM!

**Pull request merged**